server/questions.js

```diff
@@ -146,19 +146,15 @@ var validateQuestionByType = function(question, type){
 }
 
 var multipleChoiceValidator = function(question){
-	if (question.choices){
-		if (question.choices.length < 2){
-			return 'Need two or more options for Multiple Choice Question';
-		}
+	if (question.choices && question.choices.length < 2){
+		return 'Need two or more options for Multiple Choice Question';
```

Comment:
If no choices supplied, the verification would pass. We should check !question.choices || question.choices.length<2

Reply:
Also make then return true or false

Reply:
Can't return true/false, the function is returning specific status back to the user. (err message)

Reply:
You are right, my bad. Overall it looks good, I just need to double check my previous comments to make sure we are not missing anything